### PR TITLE
feat(form): form组件增加自动换行

### PR DESCRIPTION
**Pull request by TC-twwang**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/input/vars.less [hunk 1]
```diff
@@ -19,4 +19,5 @@
   --ti-mobile-textarea-title-color: var(--ti-mobile-base-color-info-normal, #333);
   --ti-mobile-textarea-showlimit-inner-height: 70px;
   --ti-mobile-textarea-showcontent-inner-height: 66px;
+  --ti-mobile-input-form-error-bg: rgba(230, 34, 34, 0.04);
```
Comment (chenxi-20) [inline src/input/vars.less]: --ti-mobile-input-form-error-bg这个变量命名不规范，需要改一下；另外，这个文件和./input/index.js文件的变量命名很多都不规范，建议有时间都整改一下，比如action-sheet里面的命名就符合规范

File: src/form-item/index.less [hunk 2]
```diff
@@ -92,6 +100,21 @@
...
 
+  &__label.is-ellipsis{
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    display: initial;
+  }
+  
+  &.is-error .is-vertical{
+    margin-bottom: 36px !important;
```
Comment (zzcr) [inline src/form-item/index.less]: 尽量不要使用!important，可以使用其他提高权重的方法

File: src/input/vars.less (hunk 1 above)
Comment (TC-twwang) [reply src/input/vars.less]: input组件已整改

File: src/form-item/index.less (hunk 2 above)
Comment (TC-twwang) [reply src/form-item/index.less]: 已删除 !important提权

**Review by zzcr — APPROVED** [review]

File: src/input/vars.less [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
-  --ti-mobile-input-height: var(--ti-mobile-base-size-height-minor, 30px);
+  --ti-mobile-input-bg-color: var(--ti-mobile-base-color-light, #fff);
+  --ti-mobile-input-line-height: var(--ti-mobile-base-size-height-minor, 30px);
   --ti-mobile-input-medium-height: var(--ti-mobile-base-size-height-medium, 42px);
   --ti-mobile-input-small-height: var(--ti-mobile-base-size-height-small, 36px);
   --ti-mobile-input-mini-height: var(--ti-mobile-base-size-height-mini, 24px);
-  --ti-mobile-input-disabled-color: var(--ti-mobile-base-color-placeholder, #999);
-  --ti-mobile-input-disabled-bgcolor: var(--ti-mobile-base-color-selected-background, #f5f5f5);
-  --ti-mobile-input-exceed-color: var(--ti-mobile-base-color-danger-normal, #f5222d);
+  --ti-mobile-input-color-disabled: var(--ti-mobile-base-color-placeholder, #999);
```
Comment (chenxi-20) [inline src/input/vars.less:7]: --ti-mobile-input-color-disabled这个命名不符合规范，因为此变量用于color或fill属性，按照命名规范，建议整改成--ti-mobile-input-text-color-disabled或--ti-mobile-input-fill-color-disabled

File: src/input/vars.less [hunk 4]
```diff
@@ -1,22 +1,23 @@
...
   --ti-mobile-input-icon-font-size: 16px;
   --ti-mobile-input-icon-close-color: var(--ti-mobile-base-color-primary-disabled, #bfbfbf);
-  --ti-mobile-textarea-count-color: var(--ti-mobile-base-color-placeholder, #999);
+  --ti-mobile-textarea-count-text-color: var(--ti-mobile-base-color-placeholder, #999);
   --ti-mobile-textarea-count-font-size: var(--ti-mobile-base-font-size, 12px);
   --ti-mobile-textarea-inner-height: 96px;
   --ti-mobile-textarea-inner-font-size: 16px;
-  --ti-mobile-textarea-inner-color: var(--ti-mobile-base-color-secondary, #666);
+  --ti-mobile-textarea-inner-text-color: var(--ti-mobile-base-color-secondary, #666);
   --ti-mobile-textarea-inner-placeholder-color: #ccc;
```
Comment (chenxi-20) [inline src/input/vars.less:17]: --ti-mobile-textarea-inner-placeholder-color也是一样的问题

File: src/input/vars.less [hunk 5]
```diff
@@ -1,22 +1,23 @@
...
   --ti-mobile-input-small-height: var(--ti-mobile-base-size-height-small, 36px);
   --ti-mobile-input-mini-height: var(--ti-mobile-base-size-height-mini, 24px);
-  --ti-mobile-input-disabled-color: var(--ti-mobile-base-color-placeholder, #999);
-  --ti-mobile-input-disabled-bgcolor: var(--ti-mobile-base-color-selected-background, #f5f5f5);
-  --ti-mobile-input-exceed-color: var(--ti-mobile-base-color-danger-normal, #f5222d);
+  --ti-mobile-input-color-disabled: var(--ti-mobile-base-color-placeholder, #999);
+  --ti-mobile-input-bg-color-disabled: var(--ti-mobile-base-color-selected-background, #f5f5f5);
+  --ti-mobile-input-exceed-text-color: var(--ti-mobile-base-color-danger-normal, #f5222d);
   --ti-mobile-input-icon-font-size: 16px;
   --ti-mobile-input-icon-close-color: var(--ti-mobile-base-color-primary-disabled, #bfbfbf);
```
Comment (chenxi-20) [inline src/input/vars.less:11]: --ti-mobile-input-icon-close-color也是类似的问题

**Pull request merged**